torch/_refs/__init__.py

```diff
@@ -424,6 +425,30 @@ def _fn(*args, **kwargs):
     return _fn
 
 
+def _make_inplace(fn):
+    """
+    Given a function with out variant (i.e. using `out_wrapper()), it returns its in-place variant
```

Comment:
Should we point the reader to the wiki faq discussing the semantics of this?

Reply:
Sure thing